src/commands/utility/requestPoint.js

```diff
@@ -27,33 +37,26 @@ module.exports = {
   async execute(interaction) {
     try {
       const { options, channel, guild } = interaction;
-
       if (!channel.isThread()) {
         return await interaction.reply({
           content: translateLanguage('requestPoint.notAThread'),
           ephemeral: true,
         });
       }
-
       const user = options.getUser('user');
       const userMessage = options.getString('reason');
-      const escapedUserId = `<@${user.id}>`;
-
-      const threadLink = `https://discord.com/channels/${guild.id}/${channel.id}`;
-      const message = translateLanguage('requestPoint.message', {
-        userId: escapedUserId,
-        reason: userMessage,
-        threadLink,
+      const message = buildRequestMessage({
+        user,
+        userMessage,
+        guild,
+        channel,
       });
-
-      const channelSend = await interaction.client.channels.fetch(
+      const adminChannel = await interaction.client.channels.fetch(
         REQUEST_POINT.discordAdminPointRequestChannel
       );
-
-      if (channelSend) {
-        await channelSend.send(message);
+      if (adminChannel) {
+        await adminChannel.send(message);
       }
-
       await interaction.reply({
         content: translateLanguage('requestPoint.success'),
         ephemeral: true,
```

Comment:
If adminChannel is not found it should stop the execution and notify the user the request point has not been successful, since it's giving a false positive to the user but the admin is not seeing the point requested, what do you think?